backend/core/database.py

```diff
@@ -845,13 +847,21 @@ def handle_hijack_ongoing_request(self, message):
                             {
                                 "key": entry[0],  # key
                                 "prefix": entry[1],  # prefix
-                                "path": entry[2],  # as_path
-                                "type": entry[3],  # type
-                                "hij_key": entry[4],
-                                "hijack_as": entry[5],
-                                "hij_type": entry[6],
+                                "origin_as": entry[2],  # origin ASN
```

Comment:
I added the following extra info on the ongoing hijack BGP updates so that they can be properly processed by the detector in case they remain hijacks.